ContriHub/settings.py

```diff
@@ -113,10 +113,10 @@
         'default': {
             'ENGINE': 'django.db.backends.postgresql_psycopg2',
             'NAME': 'contrihub_db',
-            'USER': 'contrihub_user',
```

Comment:
You needn't modify database configuration, you can use sqlite or even if you modify it reset it to initial state as it was before creating PR.